visidata/loaders/imap.py

```diff
@@ -1,11 +1,12 @@
-from visidata import vd, TableSheet, asyncthread, ColumnItem, Column, ColumnAttr, Progress
+from visidata import VisiData, vd, TableSheet, asyncthread, ColumnItem, Column, ColumnAttr, Progress
+import vdplus.api.google
```

Comment:
@saulpw Build is failing because of this line.

I'd say we can take it as-is, and figure things out on our end.